network/types/src/peer_info.rs

```diff
@@ -104,6 +104,19 @@ impl PeerInfo {
     }
 }
 
+impl From<PeerInfo> for PeerInfoView {
```

Comment:
we donot use this crate, find another way to do these.

Reply:
fixed